Formula/akamai.rb

```diff
@@ -0,0 +1,29 @@
+class Akamai < Formula
+  desc "CLI toolkit for working with Akamai's API from the command-line."
```

Comment:
`desc` should not have a "." at the end, and you can remove "from the command-line" since you've already mentioned it's a "CLI toolkit."